src/python/pants/backend/nfpm/util_rules/inject_config.py

```diff
@@ -94,6 +93,13 @@ def is_applicable(cls, target: Target) -> bool:
         """Whether to use this InjectNfpmPackageFieldsRequest implementation for this target."""
 
 
+@rule(polymorphic=True)
+async def get_inject_nfpm_package_fields(
```

Comment:
Why not just `inject_nfpm_package_fields`? "get" and "inject" sounds odd next to each other. What about `get_injected_nfpm_package_fields`?